synapse/lib/stormlib/model.py

```diff
@@ -812,3 +814,189 @@ async def _methCopyTags(self, src, dst, overwrite=False):
                 for propname, valu in tagprops.items():
                     if overwrite or not dst.hasTagProp(tagname, propname):
                         await proto.setTagProp(tagname, propname, valu) # use tag perms
+
+@s_stormtypes.registry.registerLib
+class LibModelMigrations(s_stormtypes.Lib):
+    '''
+    A Storm library for selectively migrating nodes in the current view.
+    '''
+    _storm_locals = (
+        {'name': 'itSecCpeFixup',
+         'desc': '''
+            Versions of Synapse prior to v2.169.0 did not correctly parse and
+            convert CPE strings from 2.2 -> 2.3 or 2.3 -> 2.2. This migration
+            attempts to re-normalize it:sec:cpe nodes that may be fixable.
+
+            NOTE: It is highly recommended it:sec:cpe migrations occur in a
```

Comment:
We may want to clarify/expand this.  I think this would be "test the migrations in a fork" and then run using $lib.view.list(deporder=true) { $lib.layer.get().liftByProp(it:sec:cpe) /* do stuff */ }, unless the intent is for a user to create a fork on every write layer and then merge each migration.